app/questionnaire/rules/operations.py

```diff
@@ -242,3 +242,11 @@ def evaluate_option_label_from_value(self, value: str, answer_id: str) -> str:
             label = self.renderer.render_placeholder(label_options, list_item_id=None)
 
         return label
+
+    @staticmethod
+    def evaluate_sum(*args: tuple) -> Union[int, Decimal]:
+        values_list: list = []
+        for arg in args:
+            if arg and isinstance(arg, (int, Decimal)):
+                values_list.append(arg)
+        return sum(values_list)
```

Comment:
We are a bit inconsistent in runner, most of the operations don't validate and expect the correct types.
However, happy to keep as it does guard if and when we pull from metadata etc.
```suggestion
        sum(value for value in args if isinstance(value, (int, float, Decimal)))
```

Reply:
Updated this 👍